src/test/java/org/cactoos/func/RepeatedTest.java

```diff
@@ -49,10 +49,11 @@ public void runsFuncMultipleTimes() throws Exception {
             },
             3
         );
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "",
```

Comment:
@andreoss all of those should have a clear message in the form "Must..."

Reply:
@victornoel Done